### PR TITLE
Issue 113: Implement Settings page Frontend

### PR DESCRIPTION
**Pull request by rusisolanki**
## Summary by CodeRabbit

- **New Features**
  - Introduced a new settings page with tabbed navigation for managing user profiles, passwords, and team settings.
  - Implemented a ProfileTab for users to update their personal information and manage their profile photo.

- **Style**
  - Added new styles for the settings layout and tabs to enhance visual hierarchy and improve user experience.
  - Updated styles for tab labels to ensure consistency and readability across the application.

### CODE REVIEW

File: frontend/src/scenes/settings/SettingsTabs/ProfileTab/ProfileTab.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+              <div className="button-options">
+                <p>Delete</p>
+                <button className="update">Update</button>
+              </div>
+            </div>
+          </div>
+          <div className="save-button">
+            <button className="save">Save</button>
+          </div>
+        </form>
```
[the remark this answers is not in the document]
Comment (rusisolanki) [reply frontend/src/scenes/settings/SettingsTabs/ProfileTab/ProfileTab.jsx]: Added the code

File: frontend/src/scenes/settings/SettingsTabs/SettingsTabs.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import Box from '@mui/material/Box';
+import Tab from '@mui/material/Tab';
+import TabContext from '@mui/lab/TabContext';
+import TabList from '@mui/lab/TabList';
+import TabPanel from '@mui/lab/TabPanel';
+import './SettingsTabs.css'
+import ProfileTab from './ProfileTab/ProfileTab';
+
+
+const tabLabel = ['Profile', 'Password', 'Team']
```
[the remark this answers is not in the document]
Comment (rusisolanki) [reply frontend/src/scenes/settings/SettingsTabs/SettingsTabs.jsx]: Removed the variable

File: frontend/src/scenes/settings/Settings.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+const Settings = () => {
+  return (
+    <div>
+      <HomePageTemplate>
+        <div className="settings">
+          <SettingsTabs />
+        </div>
+      </HomePageTemplate>
+    </div>
+  );
```
[the remark this answers is not in the document]
Comment (rusisolanki) [reply frontend/src/scenes/settings/Settings.jsx]: Added the tag

File: frontend/src/scenes/settings/Settings.jsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import SettingsTabs from "./SettingsTabs/SettingsTabs";
+import HomePageTemplate from "../../components/templates/HomePageTemplate";
+import './Settings.css'
+
+const Settings = () => {
+  return (
+    <div>
```
Comment (erenfn) [inline frontend/src/scenes/settings/Settings.jsx:8]: remove main and div tags

File: frontend/src/scenes/settings/Settings.css [hunk 5]
```diff
@@ -0,0 +1,6 @@
+.settings{
```
Comment (erenfn) [inline frontend/src/scenes/settings/Settings.css:1]: move this to settings  tabs css

File: frontend/src/scenes/settings/Settings.jsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+      <main>
+      <HomePageTemplate>
+        <div className="settings">
```
Comment (erenfn) [inline frontend/src/scenes/settings/Settings.jsx:11]: remove this div and move the styling to settings tabs css

**Review by erenfn — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/settings/Settings.jsx (hunk 4 above)
Comment (rusisolanki) [reply frontend/src/scenes/settings/Settings.jsx]: Resolved

File: frontend/src/scenes/settings/Settings.jsx (hunk 6 above)
Comment (rusisolanki) [reply frontend/src/scenes/settings/Settings.jsx]: Done

File: frontend/src/scenes/settings/Settings.css (hunk 5 above)
Comment (rusisolanki) [reply frontend/src/scenes/settings/Settings.css]: Done

File: frontend/src/scenes/settings/SettingsTabs/SettingsTabs.jsx [hunk 7]
```diff
@@ -15,6 +15,7 @@ export default function SettingsTabs() {
   };
 
   return (
+    <div className="settings">
```
Comment (erenfn) [inline frontend/src/scenes/settings/SettingsTabs/SettingsTabs.jsx:18]: either use div or Box. (there are pretty much the same thing) One of them is redundant

**Review by erenfn — CHANGES_REQUESTED** [review]

File: frontend/package-lock.json
Comment (erenfn) [inline frontend/package-lock.json:1]: also package-lock.json creates conflicts. Do not push changes to this file

Comment (erenfn) [reply frontend/package-lock.json:1]: You can view the list of commits that modified package-lock.json using :

- git log package-lock.json

You can set package-lock.json back to its version in commit [ID] using :

- git checkout [ID] -- package-lock.json

Comment (erenfn) [reply frontend/package-lock.json:1]: How is this going?

**Review by uparkalau — CHANGES_REQUESTED** [review]
Close this PR and open new one for the develop branch not for the master.
Please in you PR implement all recommended changes.

**Pull request closed without merging**